src/repo.rs

```diff
@@ -77,13 +79,33 @@ pub fn repo_sync(repo: &str, force: bool) -> Result<()> {
 
     loop {
         println!("Running repo sync...");
-        let cmd = Command::new("repo")
+        let repo_sync = format!("repo sync -j{}", &num_cpus::get());
+        // `script` is a Unix command that takes a copy of all output to the terminal
```

Comment:
Interesting.  Are there other places the "repo" command are called that we can do this as well?

If so we should also create a module that handles repo interaction as a whole and does this more generally there and callinto that function from here

Reply:
I think repo.rs is a good place for these functions to live.

So you can modify repo.rs to all use the "script" and add this function there as well for syncing.

Wdyt?  (what do you think?)